assets/js/util/constants.js

```diff
@@ -2,3 +2,5 @@ export const minWidth = 700;
 
 export const checkIfDevicesNotInFilter = (label) =>
   label.devices.filter((device) => device.in_xor_filter === false).length > 0;
+
+export const isMobile = window.innerWidth < 721 || window.innerHeight < 721;
```

Comment:
we may want to drop the innerHeight part now that I think about it since the mobile view wouldn't show just because height is low.

Instead we should check for desktop only from store state?